wallet/wallet-controller/src/lib.rs

```diff
@@ -200,6 +200,10 @@ impl<T: NodeInterface + Clone + Send + Sync + 'static> Controller<T> {
             .map_err(ControllerError::WalletError)
     }
 
+    pub fn get_pool_ids(&self, account_index: U31) -> Result<BTreeSet<PoolId>, ControllerError<T>> {
+        self.wallet.get_pool_ids(account_index).map_err(ControllerError::WalletError)
```

Comment:
We probably should list more information about this, including when the pool was created, etc. Pool Ids alone are just meaningless gibberish.

Reply:
Added block height and timestamp of the pool creation and the current balance retrieved from the node.